tests/test_object.h

```diff
@@ -87,6 +87,97 @@ TEST_CASE("[Object] Metadata") {
 			meta_list2.size() == 0,
 			"The metadata list should contain 0 items after removing all metadata items.");
 }
+
+TEST_CASE("[Object] IAPI methods") {
+	Object object;
+    bool *rvalid = memnew(bool(false));
+
+    object.set(StringName("script"), "script_source", rvalid);
```

Comment:
I'm not 100% sure but I think there's no need to explicitly convert the first argument to `StringName`:

```suggestion
    object.set("script", "script_source", rvalid);
```
The same applies to other assertions.